client/web/web.go

```diff
@@ -659,6 +660,7 @@ func (s *Server) serveGetNodeData(w http.ResponseWriter, r *http.Request) {
 	}
 	for _, r := range prefs.AdvertiseRoutes {
 		if r == exitNodeRouteV4 || r == exitNodeRouteV6 {
+			data.AdvertisingExitNodeApproved = routeApproved(r)
```

Comment:
I'd actually send this even if they're not currently advertising as an exit node. That way if already approved, it never shows the error state

Reply:
Ah fair, will pull this out and do a `data.AdvertisingExitNodeApproved = routeApproved(v4) || routeApproved(v6)` outside of the loop then 🙏

Reply:
Oh playing with this, I think allowed ips are actually just a subset of the advertised ones 😢

Reply:
Ah that is unfortunate 😩